controller/controller.js

```diff
@@ -87,3 +87,18 @@ exports.patchArticleVote = (req, res, next)=>{
         })
         .catch(next)
     }
+
+    exports.deleteComment = (req, res, next)=>{
+        const {comment_id} = req.params
+
+      const deleteCommentPromise = [ deleteByCommentId(comment_id)]
+       if(comment_id){
```

Comment:
This `if` isn't necessary, as your comment_id is part of your endpoint. It will always be required to reach this endpoint